src/Kunstmaan/PagePartBundle/Helper/FormWidgets/PagePartWidget.php

```diff
@@ -128,11 +128,19 @@ public function getFormErrors(FormView $formView)
     }
 
     /**
+     * @param string $template
```

Comment:
Why did you add these here?They are already in the FormWidget class (& interface)that this class inherits from...